core-api/src/main/java/com/optimizely/ab/internal/ConditionUtils.java

```diff
@@ -32,6 +32,75 @@
 import java.util.Map;
 
 public class ConditionUtils {
+
+    static public <T> Condition parseConditions(Class<T> clazz, Object object) throws InvalidAudienceCondition {
```

Comment:
Is it guaranteed that one of the other overloaded `parseConditions` methods is used if `object` is an instance of `List<Object>` or `org.json.JSONArray`?

Reply:
Yes. As, tested in the current parsing of datafiles.

Reply:
Cool, thanks!